Api/src/main/java/ranking/RankingService.java

```diff
@@ -0,0 +1,72 @@
+package ranking;
```

Comment:
패키지 구조는 
```suggestion
package playlist.server.ranking.service;
or
package playlist.server.rank.service;
```
로 하면 좋을 것 같습니다.